src/components/color-select/color-select.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+
+import styles from './color-select.module.scss';
+
+export interface ColorSelectOption {
+    id: string;
+    color: string;
+}
+
+export interface ColorSelectProps {
+    options: ColorSelectOption[];
+    selectedId: string;
+    onChange: (id: string) => void;
+    hasError?: boolean;
```

Comment:
What `hasError` is used for? To highlight that this is a required field?

Reply:
Yes
<img width="466" alt="Screenshot 2024-10-11 at 10 48 38" src="https://github.com/user-attachments/assets/5fd7dcaa-924c-495b-8be8-9f1900d47c21">